proof/invariant-abstract/CSpaceInv_AI.thy

```diff
@@ -12,31 +12,31 @@ theory CSpaceInv_AI
 imports ArchCSpaceInvPre_AI
 begin
 
-context begin interpretation Arch .
-
-requalify_consts
+arch_requalify_consts
   cap_master_arch_cap
   replaceable_final_arch_cap
   replaceable_non_final_arch_cap
   unique_table_refs
 
-requalify_facts
+(* There are multiple arch-dependent acap_rights_update_id, one for wellformed_acap,
+   one for valid_arch_cap. Prefer the latter. *)
```

Comment:
Not as part of this PR, but this sounds like something we should fix.

Reply:
I'll take a look, you're right, something is off here.

Reply:
fixed and added to PR